src/common/acl_api.c

```diff
@@ -1135,63 +1193,58 @@ daos_ace_is_valid(struct daos_ace *ace)
 					DAOS_ACL_FLAG_ACCESS_FAIL |
 					DAOS_ACL_FLAG_ACCESS_SUCCESS;
 	uint64_t	valid_perms =	DAOS_ACL_PERM_READ |
-					DAOS_ACL_PERM_WRITE;
+					DAOS_ACL_PERM_WRITE |
```

Comment:
This appears to be == cont_perms | pool_perms.   If you define macros, you can reduce some duplication, and maybe avoid future omissions.

Reply:
Good idea, I'll address this in a follow-on PR.